libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -726,6 +746,7 @@ export function Tracker(
               schema: 'iglu:com.snowplowanalytics.snowplow/web_page/jsonschema/1-0-0',
```

Comment:
I think the schema version should be `1-1-0` here.

Reply:
Yeap missed that, thanks!